lib/client/log.js

```diff
@@ -22,4 +22,6 @@ const silent = {
   warn: noop
 };
 
-module.exports = () => (window.webpackPluginServe.silent ? silent : log);
```

Comment:
this can just be:

```js
module.exports = () => (window && window.webpackPluginServe.silent ? silent : log);
```

Reply:
mmm... I'm pretty sure this will give us
```
ReferenceError: window is not defined
```
in node